app/characterization_services/mediainfo_characterization_service.rb

```diff
@@ -41,12 +41,7 @@ def characterize(save: true)
   end
 
   def duration
-    if media.model.video?
-      media.duration
-    elsif media.model.audio?
-      # MediaInfo returns audio duration in milliseconds
-      media.duration.to_f / 1000
-    end
+    media.duration.to_f / 1000
```

Comment:
I guess MediaInfo is more internally consistent now? It would be nice to keep the comment about milliseconds to explain the number `1000`